pkg/cmd/cli/restore/logs.go

```diff
@@ -35,15 +37,14 @@ func NewLogsCommand(f client.Factory) *cobra.Command {
 	c := &cobra.Command{
 		Use:   "logs RESTORE",
 		Short: "Get restore logs",
+		Args:  cobra.ExactArgs(1),
 		Run: func(c *cobra.Command, args []string) {
-			if len(args) != 1 {
-				err := errors.New("restore name is required")
-				cmd.CheckError(err)
-			}
+			l := NewLogsOptions()
+			cmd.CheckError(l.Complete(args))
+			cmd.CheckError(l.Validate(f))
```

Comment:
Unlike it's done in create.go, I'm calling Complete() before Validate(), because it sort of made sense to first let complete populate the restore name and then do the validations. Let me know if you'd want this done the other way round.

Reply:
Yes, let's plan to switch to 1) complete 2) validate 3) run. I'll file an issue to do this everywhere.

Reply:
#411